packages/realm-server/package.json

```diff
@@ -89,6 +89,7 @@
     "start:staging": "./scripts/start-staging.sh",
     "start:development": "./scripts/start-development.sh",
     "start:production": "./scripts/start-production.sh",
+    "start:services-for-matrix-tests": "./scripts/start-services-for-matrix-tests.sh",
```

Comment:
Is this now preferable over `start-without-matrix` to run matrix tests?

Reply:
yeah, it's gonna run only the realms necessary for matrix tests. you can still use `start:all` if you want--but you'll be running realms you dont need (and you'll have to turn off synapse, since playwright needs to control that docker container).